service/src/main/java/io/spaship/operator/rest/ApiRootResource.java

```diff
@@ -10,23 +11,35 @@
 import javax.ws.rs.Path;
 import javax.ws.rs.Produces;
 import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Request;
 import java.util.List;
 
 @Path("/api")
 @Produces(MediaType.APPLICATION_JSON)
 public class ApiRootResource {
 
-    @ConfigProperty(name = "quarkus.http.root-path")
```

Comment:
Isn't `@ConfigProperty(name = "quarkus.http.root-path")` field more readable rather than Constructor injection ?
You can use `@Inject EventSourcingEngine eventSourcingEngine` field as well.